storage/innobase/xtrabackup/src/keyring_plugins.cc

```diff
@@ -71,7 +109,25 @@ static void init_plugins(int argc, char **argv) {
   memcpy(t_argv, argv, sizeof(char *) * t_argc);
 
   mysql_optional_plugins[0] = 0;
-  mysql_mandatory_plugins[0] = 0;
+
+  /* initialize only daemon_keyring_proxy_plugin */
+  if (initialize_keyring_proxy) {
```

Comment:
Do we need this bool variable? ie. Do we call init_plugins() twice?

Reply:
Not required. Removed